drivers/adc/adc_stm32.c

```diff
@@ -90,15 +96,15 @@ static const uint32_t table_resolution[] = {
 #endif
 };
 
-#define SMP_TIME(x, y)		LL_ADC_SAMPLINGTIME_##x##CYCLE##y
+#define SMP_TIME(x, y)          LL_ADC_SAMPLINGTIME_##x##CYCLE##y
 
 /*
  * Conversion time in ADC cycles. Many values should have been 0.5 less,
  * but the adc api system currently does not support describing 'half cycles'.
  * So all half cycles are counted as one.
  */
 #if defined(CONFIG_SOC_SERIES_STM32F0X) || defined(CONFIG_SOC_SERIES_STM32F1X)
-static const uint16_t acq_time_tbl[8] = {2, 8, 14, 29, 42, 56, 72, 240};
+static const uint16_t acq_time_tbl[8] = { 2, 8, 14, 29, 42, 56, 72, 240 };
```

Comment:
These changes seem odd to me. Even on upstream Zephyr there's no spaces there. Does the format fail if those spaces are removed?

Comment applies for similar cases below.

Reply:
That's exactly why I made this comment. As mentioned the script isn't perfect and shouldn't be taken as the truth. The upstream Zephyr also looks different, and we intend to push these changes there.

Reply:
formatting restored